spec/matcher_spec.rb

```diff
@@ -1,18 +1,20 @@
 require 'chefspec'
 require 'chefspec/berkshelf'
 
-describe 'chocolatey_specs::default' do
+RSpec.describe 'chocolatey_specs::default' do
```

Comment:
never mind. I see this is the "new" way of doing things and it makes sense. Just wasn't used to seeing it.